src/mechanics/elements/Element.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include "mechanics/elements/ElementInterface.h"
+#include "mechanics/nodes/DofContainer.h"
+
+namespace NuTo
+{
+class Element
+{
+public:
+    Element(const ElementInterface& coordinateElement)
+        : mCoordinateElement(coordinateElement)
+    {
+    }
+    Element(const ElementInterface& coordinateElement, DofContainer<const ElementInterface*> dofElement)
+        : mCoordinateElement(coordinateElement)
+        , mDofElements(dofElement)
+    {
+    }
+
+    void AddElement(const DofType& dofType, const ElementInterface& dofElement)
+    {
```

Comment:
Naming implies that an Element object can add another Element object to itself... That does not make sense.